tagmanager/ctags/powershell.c

```diff
@@ -0,0 +1,658 @@
+/*
+*   Copyright (c) 2015, Enrico Tröger <enrico.troeger@uvena.de>
+*
+*   Loosely based on the PHP tags parser since the syntax is somewhat similar
+*   regarding variable and function definitions.
+*
+*   This source code is released for free distribution under the terms of the
+*   GNU General Public License.
+*
+*   This module contains code for generating tags for Windows PowerShell scripts.
+*/
+
+/*
+*   INCLUDE FILES
+*/
+#include "general.h"  /* must always come first */
+#include "main.h"
+#include "parse.h"
+#include "read.h"
+#include "vstring.h"
+#include "keyword.h"
+#include "entry.h"
+#include <string.h>
+
+#define SCOPE_SEPARATOR ":"
+
+
+#define ACCESS_UNDEFINED "undefined"
```

Comment:
looking at how it's used, why not making this simply `NULL`?  it would avoid having to compare string literals